pkg/certs/cert-inspection/certgraphanalysis/options.go

```diff
@@ -1,38 +1,80 @@
 package certgraphanalysis
 
 import (
+	"context"
+	"fmt"
 	"strings"
 
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 )
 
 type certGenerationOptions struct {
-	rejectConfigMap configMapFilterFunc
-	rejectSecret    secretFilterFunc
+	rejectConfigMap  configMapFilterFunc
+	rejectSecret     secretFilterFunc
+	rewriteConfigMap configMapRewriteFunc
+	rewriteSecret    secretRewriteFunc
 }
 
 var (
 	SkipRevisioned = &certGenerationOptions{
 		rejectConfigMap: func(configMap *corev1.ConfigMap) bool {
-			if isRevisioned(configMap.OwnerReferences) {
-				return true
-			}
-			return false
+			return isRevisioned(configMap.OwnerReferences)
 		},
 		rejectSecret: func(secret *corev1.Secret) bool {
-			if isRevisioned(secret.OwnerReferences) {
-				return true
+			return isRevisioned(secret.OwnerReferences)
+		},
+	}
+	RewriteNames = &certGenerationOptions{
```

Comment:
This needs to be selective.  So multiple instances like `RewriteNodeName`, `RewriteNodeIP` so that it spans the logical name substitutions.

Reply:
Done, renamed into RewriteNodeIP